torch/fx/passes/operator_support.py

```diff
@@ -2,10 +2,11 @@
 
 import torch
 import torch.fx
+from torch.fx._compatibility import compatibility
 
 from .tools_common import get_node_target, CALLABLE_NODE_OPS
 
-
+@compatibility(is_backward_compatible=False)
```

Comment:
cc @842974287 FYI I'm conservatively setting backward-compatibility for the APIs in the file to `False`. Please feel free to flip this to `True` if/when you feel comfortable providing stability guarantees for the API.